puppet-squid/src/main/java/com/iadams/sonarqube/puppet/api/PuppetGrammar.java

```diff
@@ -198,15 +193,7 @@ public static void grammar(LexerfulGrammarBuilder b) {
 
         b.rule(PARAMS).is(b.optional(
                 PARAM,
-                b.zeroOrMore(COMMA, PARAM))).skip();
```

Comment:
@iwarapter We have to make a decision on this because it's blocking as lots of rules check parameter nodes. I currently have to merge this branch to other local branches to continue developing. And I can't create other pull requests for now for my new developments based on this grammar update.